docs/_config.yml

```diff
@@ -1,4 +1,6 @@
 # Copyright 2018 VMware, Inc.
```

Comment:
Because you are changing a significant part of that file maybe it's a good idea to update the year of the Copyright.

Reply:
Good catch, thanks.